src/interface/report/PlayerSelection/index.tsx

```diff
@@ -1,22 +1,23 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
 import SPECS from 'game/SPECS';
 import ROLES from 'game/ROLES';
+import { CombatantInfoEvent } from 'parser/core/Events';
 
 import PlayerTile from './PlayerTile';
 import './PlayerSelection.scss';
 
-const ROLE_SORT_KEY = {
+const ROLE_SORT_KEY: {[key: string]: number} = {
   [ROLES.TANK]: 0,
   [ROLES.HEALER]: 1,
   [ROLES.DPS.MELEE]: 2,
-  [ROLES.DPS.RANGED]: 2,
+  //I'm not sure if this is a bug or intended behaviour, But if this is 2 then there is no sort based or range/melle only on dps.
+  [ROLES.DPS.RANGED]: 3,
```

Comment:
It was intended to be 2. We experimented with grouping melee and ranged, and found it more intuitive to just group all DPS together. Maybe leave a comment to clarify that?

Reply:
Thanks for your response, updated.